include/signalrclient/connection.h

```diff
@@ -29,6 +29,10 @@ namespace signalr
 
         SIGNALRCLIENT_API ~connection();
 
+        connection(const connection&) = delete;

```

Comment:
This needs to be discussed when we review the API. Prevents from cycles by capturing the connection in the `message_received` callback but requires passing by reference or using a pointer. (To capture in the `message_received` the user needs to actually use a `weak_ptr`)